Notebooks/grid_search_GD.ipynb

```diff
@@ -38,14 +38,14 @@
   },
```

Comment:
<p>Good idea to use one placement for all time windows as a baseline - could we just use that for the other metrics as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AndOleAnd/Capstone_N_A_P/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>